gem/templates/chhaajaa/core/tags/hero_article.html

```diff
@@ -99,5 +99,93 @@ <h4 class="teaser-stacklist__subtitle">{{ article.subtitle|smarttruncatechars:15
     {% surveys_list_for_pages page=article %}
     {% include "comments/comment_block.html" %}
   {% endfor %}
+  {% block extra_js %}
+    <script type="text/javascript">
+      var frm = $('#reaction-questions__form');
+      function set_choice(choice) {
+        if (!$('#' + choice).attr('disabled') == true){
+          $.ajax({
+            type: frm.attr('method'),
+            url: frm.attr('action'),
+            data: {
+              csrfmiddlewaretoken: "{{ csrf_token }}",
+              choice: choice,
+              ajax: "True"
+            },
+            success: function () {
+              $( ".reaction-questions__item" ).each(function( index ) {
+                $(this).attr('disabled', false);
+              });
+              $("#" + choice).attr('disabled', true);
+            }
+          })
+        }
+      };
+
+      document.getElementById("id_comment").addEventListener("mouseup", function(e) {
+        if(document.activeElement === e.target) {
+          e.target.closest(".comments__form-post").setAttribute("style","border: 5px solid #72f3a2; outline: none; padding: 10px; border-radius: 5px;");
+        } else {
+          console.log(document.activeElement);
+          //e.target.closest(".comments__form-post").setAttribute("style");
+        }
+      });
+
+      var elems  = document.getElementsByClassName("choice-group");
+      for(var i = 0; i < elems.length; i++) {
+        console.log(elems[i]);
```

Comment:
Did you forget to remove these print statements? or are they needed for something?